core/src/services/web-components.js

```diff
@@ -324,6 +327,26 @@ class WebComponentSvcClass {
       });
     });
   }
+
+  /**
+   * Get stored user settings for a specific user settings group
+   * @param {Object} wc node object definition
+   * @returns a promise that gets resolved with the stored user settings for a specific user settings group.
+   */
+  getUserSettingsForWc(wc) {
+    return new Promise(resolve => {
+      if (wc.userSettingsGroup) {
+        const userSettingsGroupName = wc.userSettingsGroup;
+        LuigiConfig.readUserSettings().then(storedUserSettingsData => {
+          const hasUserSettings =
+            userSettingsGroupName && typeof storedUserSettingsData === 'object' && storedUserSettingsData !== null;
+
+          let userSettings = hasUserSettings ? storedUserSettingsData[userSettingsGroupName] : null;

```

Comment:
```suggestion
          const userSettings = hasUserSettings ? storedUserSettingsData[userSettingsGroupName] : null;
```
Maybe const since its not being reassigned